### PR TITLE
Add AI and poetry blog post

### PR DESCRIPTION
**Pull request by mosabua**

### CODE REVIEW

File: _posts/2022-12-06-poetry-ai.md [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+time trying. If you are interested to learn more you can look at the [blog post
+on Ars
+Technica](https://arstechnica.com/information-technology/2022/11/openai-conquers-rhyming-poetry-with-new-gpt-3-update/)
+or go and try it yourself [on the
+playground](https://beta.openai.com/playground/).
+
+Enjoy while we are heading back to [working on Trino pull
+requests](https://github.com/trinodb/trino/pulls) and other code now.
+
+Macros and Martin
```
Comment (martint) [inline _posts/2022-12-06-poetry-ai.md:52]: Typo: s/Macros/Marcos

Comment (bitsondatadev) [reply _posts/2022-12-06-poetry-ai.md:52]: Now we have a nickname for @mtraverso3

File: _posts/2022-12-06-poetry-ai.md [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+optimizer, or some other tricky problem or code, you run out of steam and need a
+distraction. Your brain will quietly and in the background still work on the
+problem, while you get to read or do something interesting.
+
+That's when we found out more about GPT-3 and the recent improvements on the
+OpenAI project. So we had to try.
+
+<!--more-->
+
+Its pretty amazing that you can feed the AI a request just like we did:
```
Comment (mtraverso3) [inline _posts/2022-12-06-poetry-ai.md:19]:
```suggestion
It's pretty amazing that you can feed the AI a prompt just like we did:
```
Maybe use `prompt` instead of `request` as it is more commonly referred to as this.

File: _posts/2022-12-06-poetry-ai.md [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+Gathers data of different types,
+Helping you make decisions for all.
+So Trino is an integration tool,
+That connects data for you,
+It'll ease the burden of data sync,
+Giving you more time to do what you do.
+```
+
+Cool, huh? I am sure we could not have done better even after a long time
+trying. Have a look at another question:
```
Comment (mtraverso3) [inline _posts/2022-12-06-poetry-ai.md:43]:
```suggestion
trying. Have a look at another prompt:
```
Same suggestion as above!

File: _posts/2022-12-06-poetry-ai.md [hunk 4]
```diff
@@ -0,0 +1,76 @@
...
+If you are interested to learn more you can look at the [blog post
+on Ars
+Technica](https://arstechnica.com/information-technology/2022/11/openai-conquers-rhyming-poetry-with-new-gpt-3-update/)
+or go and try it yourself [on the
+playground](https://beta.openai.com/playground/).
+
+Enjoy while we are heading back to [working on Trino pull
+requests](https://github.com/trinodb/trino/pulls) and other code now.
+
+Macros and Martin
```
Comment (mtraverso3) [inline _posts/2022-12-06-poetry-ai.md:76]:
```suggestion
Marcos and Martin
```
Typo

File: _posts/2022-12-06-poetry-ai.md [hunk 5]
```diff
@@ -0,0 +1,76 @@
+---
+layout: post
+title: "AI and poetry meet Trino"
+author: Martin Traverso, Marcos Traverso
+excerpt_separator: <!--more-->
+image: /assets/trino-light.png
+---
+
+Sometimes working in the depth of the query engine core, the planner, the
```
Comment (colebow) [inline _posts/2022-12-06-poetry-ai.md:9]:
```suggestion
Sometimes when working in the depth of the query engine core, the planner, the
```

File: _posts/2022-12-06-poetry-ai.md [hunk 6]
```diff
@@ -0,0 +1,76 @@
...
+optimizer, or some other tricky problem or code, you run out of steam and need a
+distraction. Your brain will quietly and in the background still work on the
+problem, while you get to read or do something interesting.
```
Comment (colebow) [inline _posts/2022-12-06-poetry-ai.md:12]:
```suggestion
distraction. Your brain will quietly work on the problem in the background
while you get to read or do something interesting.
```

File: _posts/2022-12-06-poetry-ai.md [hunk 7]
```diff
@@ -0,0 +1,76 @@
...
+
+That's when we found out more about GPT-3 and the recent improvements on the
+OpenAI project. So we had to try.
```
Comment (colebow) [inline _posts/2022-12-06-poetry-ai.md:15]: "That's when" feels a little jarring. Maybe "one of these times," or "during one break, we-" to make it more clear you're transitioning from talking about a general phenomenon to a specific story.

File: _posts/2022-12-06-poetry-ai.md (hunk 3 above)
Comment (colebow) [inline _posts/2022-12-06-poetry-ai.md:43]: This might be the bachelor's in English talking, but we definitely could've done better, and I'd probably rephrase this to be a little less grandiose of a claim...

File: _posts/2022-12-06-poetry-ai.md [hunk 8]
```diff
@@ -0,0 +1,76 @@
...
+```
+Write a short rhyming poem explaining Trino, the SQL query engine in complex
+and accurate terms.
+```
+
+And the compelling result:
+
+```
+Trino is a query engine that does more than just run,
+It’s an SQL engine with performance that stuns.
```
Comment (colebow) [inline _posts/2022-12-06-poetry-ai.md:54]: "an SQL"... the bot has chosen its side, and it has chosen wrong. 😆

File: _posts/2022-12-06-poetry-ai.md [hunk 9]
```diff
@@ -0,0 +1,76 @@
...
+It can quickly analyze any course.
+It’s flexible, reliable, and fast,
+And is designed to make your queries last.
+It can handle large volumes of data with ease,
+Making the most complex queries a breeze.
+```
+
+If you are interested to learn more you can look at the [blog post
+on Ars
+Technica](https://arstechnica.com/information-technology/2022/11/openai-conquers-rhyming-poetry-with-new-gpt-3-update/)
```
Comment (colebow) [inline _posts/2022-12-06-poetry-ai.md:69]: I'd just squash the entire link text & link onto one line, it's more wide but also more readable that way.

**Comment by mosabua** [comment]
All addressed .. I will delay publishing till tomorrow morning to give you a chance to look again but ultimately I would like to publish then..

**Review by colebow — APPROVED** [review]
